solution.cpp

```diff
@@ -1,19 +1,135 @@
 /* ----------------------------------------------------------------------------
- * Copyright &copy; 2015 TODO(your name) <TODO(your email)>
+ * Copyright &copy; 2015 Eugine Suh <eugine@csu.fullerton.edu>
  * Released under the [MIT License] (http://opensource.org/licenses/MIT)
  * ------------------------------------------------------------------------- */
 
 /**
- * TODO: program description
+ * TODO: This program changes integer to roman numeral
  */
 
 // TODO: `#include`s and function prototypes
+#include<iostream>
+using std::cout;
+using std::cin;
+using std::endl;
 
 // ----------------------------------------------------------------------------
 
-int main() {
+int main() 
+{
     // TODO
+double num;
+int intnum, m, d, c, l, x, v, i, n;
+char answer = 'y';
 
+while (answer == 'y')
+{
+    cout<<"Please enter an integer "; 
+    cin >>num;
+    intnum = (int)num;
```

Comment:
c style casting dangerous here -- `num` should probably just be an `int` to start with